jdk/make/closed/autoconf/custom-hook.m4

```diff
@@ -309,28 +309,31 @@ AC_DEFUN_ONCE([OPENJ9_THIRD_PARTY_REQUIREMENTS],
   AC_ARG_WITH(freemarker-jar, [AS_HELP_STRING([--with-freemarker-jar],
     [path to freemarker.jar (used to build OpenJ9 build tools)])])
 
-  if test "x$with_freemarker_jar" == x ; then
-    printf "\n"
-    printf "The FreeMarker library is required to build the OpenJ9 build tools\n"
-    printf "and has to be provided during configure process.\n"
-    printf "\n"
-    printf "Download the FreeMarker library and unpack it into an arbitrary directory:\n"
-    printf "\n"
-    printf "wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker-2.3.8.tar.gz\n"
-    printf "\n"
-    printf "tar -xzf freemarker-2.3.8.tar.gz\n"
-    printf "\n"
-    printf "Then run configure with '--with-freemarker-jar=<freemarker_jar>'\n"
-    printf "\n"
-
-    AC_MSG_NOTICE([Could not find freemarker.jar])
-    AC_MSG_ERROR([Cannot continue])
-  fi
+  FREEMARKER_JAR=
+  if test "x$with_cmake" == x ; then
```

Comment:
This doesn't seem to work:
```
$ bash configure --with-boot-jdk=/home/keithc/space/bootjdk8 --with-cmake
[...]
The FreeMarker library is required to build the OpenJ9 build tools
and has to be provided during configure process.
[...]
```

Reply:
Adding `--with-freemarker-jar=/home/keithc/space/freemarker.jar` gets further, but still fails:
```
make[3]: *** No rule to make target '/home/keithc/space/jdk08/build/linux-x86_64-normal-server-release/vm/j9ddr.dat', needed by '/home/keithc/space/jdk08/build/linux-x86_64-normal-server-release/jdk/ddr/classes.done'.  Stop.
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [openj9.ddr-gen] Error 2
/home/keithc/space/jdk08/closed/make/Images.gmk:48: recipe for target 'openj9.ddr-gen' failed
```

Reply:
Looks like I forgot to add the generated configure script